### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by soojison**
fix travis so that it finally passes for the first time in the history of this entire project!

### CODE REVIEW

**Comment by schmitzc** [comment]
:fireworks: Nice work!

**Comment by soojison** [comment]
Yep, I am looking up how to set up cucumber test on travis-ci currently!

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-  - DB=sqlite
-  - DB=mysql
-  - DB=postgresql
+services:
+  - elasticsearch
+before_script:
+  - sleep 10
 script:
   - RAILS_ENV=test bundle exec rake db:migrate --trace
   - bundle exec rake db:test:prepare
```
Comment (schmitzc) [inline .travis.yml]: Add a line here to run the cucumber specs:

```
- bundle exec rake cucumber
```

**Comment by schmitzc** [comment]
> I've been running bundle exec cucumber. What's the difference between that and bundle exec rake cucumber?

They both do the same thing, but the `cucumber-rails` gem adds a Rake task for running the Cucumber tests. https://github.com/cucumber/cucumber/wiki/Ruby-on-Rails

**Comment by schmitzc** [comment]
:+1:  Merge it. The tests were already failing, so you didn't break anything that wasn't already broken.

**Pull request merged**